DESCRIPTION

```diff
@@ -13,7 +13,7 @@ Authors@R:
    )
 Depends: R (>= 3.5.0)
 Imports: data.table, curl, httr, jsonlite, oai, wpp2017, countrycode,
-        stringr, XML, lubridate
+        stringr, XML, lubridate, fields
```

Comment:
do we need `grDevices` here?

Reply:
The matrix_plot function does use of the function heat.colors() fuction from grDevices. This package seems loaded by default, so I don't know whether we should specify it here?

Reply:
I think it needs to be added so that it's installed as a dependency for those that don't have `ggplot2` installed.